parsel/selector.py

```diff
@@ -76,6 +100,15 @@ def extract_first(self, default=None):
 
 
 class Selector(object):
+    """
+    :class:`Selector` allows you to select parts of an XML or HTML text using CSS
+    or XPath expressions and extract data from it.
+
+    ``text`` is utf-8 encoded text (unicode object in Python 3 or str in Python 3)
```

Comment:
This statement is contradictory: utf8-encoded text is bytes by definition, it is an opposite of Py2 unicode / Py 3 str.